contracts/oracles/BinanceOracle.sol

```diff
@@ -96,20 +108,20 @@ contract BinanceOracle is AccessControlledV8, OracleInterface {
             symbol = "WBETH";
```

Comment:
Now that we are using the symbol only to get the config, couldn't we remove these `if` statements (for WBNB and wBETH) and just add the needed entries in the `maxStalePeriod` mapping in the VIP?